### PR TITLE
docs: Add fetch feed example to withDataAtom

### PR DESCRIPTION
**Pull request by romadryud**
Hi, this example shows how to add data you've fetched to data you've fetched before.

### CODE REVIEW

File: docs/src/pages/packages/async.md [hunk 1]
```diff
@@ -63,6 +63,19 @@ export const fetchList = reatomAsync((ctx) => fetch('...'), 'fetchList').pipe(
 onConnect(fetchList.dataAtom, fetchList)
 ```
 
+### Adding data you've fetched to data you've fetched before
```
Comment (artalar) [inline docs/src/pages/packages/async.md]: Please, move it to `/pages/packages/async.md`, the docs/packages folder is autogenerated

File: docs/src/pages/packages/async.md [hunk 2]
```diff
@@ -63,6 +63,19 @@ export const fetchList = reatomAsync((ctx) => fetch('...'), 'fetchList').pipe(
...
+
+```ts
+import { reatomAsync, withDataAtom } from '@reatom/async'
+import { onConnect } from '@reatom/hooks'
+
+export const fetchFeed = reatomAsync((ctx) => fetch('...'), 'fetchFeed').pipe(
+  withDataAtom([], (_ctx, payload, state) => state.concat(payload)),
+)
+
+onConnect(fetchFeed.dataAtom, fetchFeed)
```
Comment (artalar) [inline docs/src/pages/packages/async.md]:
```suggestion
import { reatomAsync, withDataAtom } from '@reatom/async'

export const fetchFeed = reatomAsync((ctx) => fetch('...'), 'fetchFeed').pipe(
  withDataAtom([], (ctx, payload, state) => state.concat(payload)),
)
```

File: docs/src/pages/packages/async.md (hunk 1 above)
Comment (romadryud) [reply docs/src/pages/packages/async.md]: Done ✅

File: docs/src/pages/packages/async.md (hunk 2 above)
Comment (romadryud) [reply docs/src/pages/packages/async.md]: Done ✅

Comment (romadryud) [reply docs/src/pages/packages/async.md]: Done ✅

**Pull request merged**